documentation/builders/update.md

```diff
@@ -12,6 +12,98 @@ Restore your old files after the new installation was successful and check if ne
 $ diff shared/settings/jukebox.yaml resources/default-settings/jukebox.default.yaml
 ```
 
+## Manually upgrade to the latest version
+
+> [!CAUTION]
+> For optimal system updates, it is strongly recommended to utilize the upgrade feature when transitioning to the next version. Manual updates may necessitate specific migration steps and, if overlooked, could result in system failure.
+
+If you only want to update a few recent commits, this following explanation outlines the steps to do so
```

Comment:
```suggestion
If you only want to update a few recent commits, this following explanation outlines the steps to do so.
```

Reply:
Should this be fixed, as it is not yet included?

Reply:
Yes. Will refer to the work that's not done yet